relay_messages.go

```diff
@@ -228,6 +238,40 @@ func (f lazyCallReq) Arg2Iterator() (arg2.KeyValIterator, error) {
 	return arg2.NewKeyValIterator(f.Payload[f.arg2StartOffset:f.arg2EndOffset])
 }
 
+func (f *lazyCallReq) Arg2Append(key, val []byte) error {
+	arg3 := bufPool.Get().(*bytes.Buffer)
+	arg3.Write(f.Payload[f.arg2EndOffset:])
```

Comment:
the `Payload` field is confusing as it ends up being the whole 64kb - header, even in the case that the payload is actually smaller than 64kb. You probably want `SizedPayload()` to get just the parts of the frame that are "set"

(unfortunate API)